Framework/RemoteJobManagers/src/SimpleJSON.cpp

```diff
@@ -663,7 +663,7 @@ string readUntilCloseChar(istream &istr) {
           "Stream unexpectedly ended without a closing char.");
     }
 
-    if ((value.size() > 0) ||
+    if ((!value.empty()) ||
```

Comment:
This file seems to be (almost) exact copy of `Framework/RemoteAlgorithms/src/SimpleJSON.cpp` and thus has the same readability issue here.